source/reference/minio-mc/mc-replicate-diff.rst

```diff
@@ -0,0 +1,113 @@
+.. _minio-mc-replicate-diff:
+
+=====================
+``mc replicate diff``
+=====================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+.. mc:: mc replicate diff
+
+Description
+------
```

Comment:
need to fill this decorator the rest of the way

Reply:
fixed